addons/web/static/tests/views/list_tests.js

```diff
@@ -4194,6 +4194,50 @@ QUnit.module('Views', {
 
         list.destroy();
     });
+
+    QUnit.test('readonly boolean in editable list is readonly', function (assert) {
+        assert.expect(6);
+
+        var list = createView({
+            View: ListView,
+            model: 'foo',
+            data: this.data,
+            arch: '<tree editable="bottom">'
+                      + '<field name="foo"/>'
+                      + '<field name="bar" modifiers="{&quot;readonly&quot;: [[&quot;foo&quot;, &quot;!=&quot;, &quot;yop&quot;]]}"/>'
+                  +'</tree>',
+        });
+
+        // clicking on disabled checkbox with active row does not work
+        var $disabledCell = list.$('.o_data_cell:last-child').eq(1);
```

Comment:
could you write in the selector which row you're clicking on? It helps to understand the test
Ex: `list.$('.o_data_row:nth(1) .o_data_cell:last-child');'